solutions/요세푸스문제.js

```diff
@@ -0,0 +1,27 @@
+const INPUT_PATH = "../inputs/요세푸스문제.txt";
+
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : INPUT_PATH;
+const [N, K] = fs.readFileSync(filePath).toString().trim().split(" ");
+const result = [];
+
+function solution(N, K) {
+  const numArray = new Array(Number(N)).fill(0).map((value, index) => {
+    value = index + 1;
+
+    return value;
+  });
```

Comment:
```suggestion
  const numArray = Array.from({ length: N }, (_, index) => index + 1); 
```
이렇게 한다면 `map()`을 쓰지 않고도 배열을 만들 수 있을 것 같습니다 😉